### PR TITLE
Added contact us page

### LINKED ISSUE #8: Contact us page
## Purpose
A lot of people are new to Discord and don't know how to use it.  We should have a page that demonstrates how to contact our server staff with instructions and screenshots of using Modmail to inform us about errors.

### Is your feature request related to a problem? Please describe.
We get lots of users who are new to Discord every month.  Let's give them in-depth instructions on our site instead of on Discord.

## Proposed Solution
Have at it!

### Describe alternatives you've considered (optional)
The alternative is to have the instructions on Discord, but I think the website is a better medium for longform instructions.

## Acceptance Criteria
- [ ] Page should include features discussed above, and any others you think would be useful for contacting staff, etc..
- [ ] If you can wireframe, a screenshot of that would be helpful!

### PR DESCRIPTION
**Pull request by lyallstewart**
## What issue is this solving?

Closes #8 

### Description

Provides information on how to properly contact server staff through the modmail bot.

### Please make sure you've attempted to meet the following coding standards

- [X] Code has been tested and does not produce errors
- [X] Code is readable and formatted
- [X] There isn't any unnecessary commented-out code

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <title>Contacting Staff</title>
+    </head>
+    <body>
+        <h1>How to contact server staff?</h1>
+
+        <p>If there is an isse that requires staff attention, please do not ping or direct message a staff member to report it as this is against server rules and may
```
Comment (Victoria-DR) [inline contact.html:13]: Typo: `isse` -> `issue`

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    </head>
+    <body>
+        <h1>How to contact server staff?</h1>
+
+        <p>If there is an isse that requires staff attention, please do not ping or direct message a staff member to report it as this is against server rules and may 
+            result in a warn. Instead please use the ModMail bot with the method below to bring any issues to the mod team's attention.
+        </p>
+
+        <h2>The ModMail Bot:</h2>
+        <p>To report any issues you can you the ModMail bot to open a ticket with server staff.</p>
```
Comment (Victoria-DR) [inline contact.html:18]: `you can you` -> `you can use`

File: contact.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+        <ol>
+            <li>
+                <h3>Send a message containing your issue to the bot</h3>
+                <img src="src/img/modmail0.png">
+                <img src="src/img/ModMail1.png">
```
Comment (Victoria-DR) [inline contact.html:24]: Should use `modmail1.png` for consistency (lowercase).

File: contact.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            </li>
+            <li>
+                <h3>Select the Codecademy Community server by reacting with the appropriate number</h3>
+                <p>(You may have to use the arrow reactions to scroll if you are in a large number of servers)</p>
+            </li>
+            <li>
+                <h3>Sent!</h3>
+                <p>Your message has now been sent to server staff.</p>
+                <img src="src/img/ModMail2.png">
```
Comment (Victoria-DR) [inline contact.html:33]: Should use `modmail2.png` for consistency (lowercase).

File: contact.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+            </li>
+        </ol>
+
+        <h2>If the ModMail bot is offline:</h2>
+        <p> If the ModMail bot is offline and you need to report a serious violation of server rules, you can ping an online Admin or Moderator to bring our attention to the violation. However, please reserve this only for when the ModMail bot is down or if the violation requires immediate attention.</p>
```
Comment (Victoria-DR) [inline contact.html:38]: I think the "if the violation requires immediate attention" part should be placed in the `How to contact server staff?` section. You can also include an example, like scam links and mentions being spammed in all the channels.

**Review by Victoria-DR — COMMENTED** [review]
This is awesome, thanks! I added some comments for some minor changes.

**Comment by lyallstewart** [comment]
Thanks for the feedback @Victoria-DR !

I've updated those changes 😄

**Review by Victoria-DR — APPROVED** [review]
Good to go, thanks!

**Review by aedwardg — APPROVED** [review]
This is a great start to this page @lyallstewart! Thanks!

The only thing I would change is the name of the directory containing the `img` folder.  Right now it's called `src`, but the html file is outside of it.  This isn't a deal breaker for me, but I'd prefer that _if_ we have an `src` folder, all our source code for the website be inside it.  I think a more appropriate name for the _current_ `src` directory would probably be `assets` since it will likely contain just images and css files.

Let me know what you think, and if you agree we can either change it in this PR or another one 🙂

**Comment by lyallstewart** [comment]
Hi @aedwardg ,

Thanks for the review! I think that's definitely a more sensible name, I've just pushed a commit with folder renamed. 

Now that it's got the two approving reviews, am I able to merge the pull request or is there another review step? (sorry if it's a silly question - I'm a bit new to open source projects! 😀 )

**Comment by aedwardg** [comment]
@lyallstewart typically on open source projects (and work repos as well) there will be a couple of people who have permissions to merge PRs if the changes are going to be deployed. 

The way we've been doing it is that all organization members can merge to our non-deployment primary branch but only 2 or three of us have perms to merge those changes into production.

Because this project isn't as sensitive as the bot, however, we just deploy straight from main and I don't think I put any special restrictions on merging PRs other than 1 required review. 

So...go head and push the big green button so you can merge your first PR! :tada:

**Comment by lyallstewart** [comment]
Thanks @aedwardg !

I don't think I have permission to merge, would you be able to do it instead 😃 ?
image

**Comment by lyallstewart** [comment]
Yay, my first PR 🥳 !

**Pull request merged**